src/components/DocumentLayouts.js

```diff
@@ -1,60 +1,44 @@
 import React from 'react'
 
 import {
-  Text,
-  Page,
-  View,
-  Document
+    Text,
+    Page,
+    View,
+    Document
 } from '@react-pdf/renderer'
 
 // @TODO i don't like this approach, but it looks simple for now
 
 import MainLayout from './LayoutTypes/MainLayout'
 
 // import styles from './styles'
-import BulletListStyles from './LayoutStyles/BulletList'
-import RowViewStyles from './LayoutStyles/RowViewStyles'
-import ColumnViewStyles from './LayoutStyles/ColumnView'
+import { BulletList as BulletListStyles, ColumnView as ColumnViewStyles, RowView as RowViewStyles } from './LayoutTypes/LayoutStyles'
```

Comment:
I'm 100% sure that this version right now wouldn't work.
looks like right idea, but ... take more time and find out what exactly is wrong